rocAL/rocAL/source/rali_api_augmentation.cpp

```diff
@@ -1647,7 +1647,7 @@ RALI_API_CALL raliCropMirrorNormalize(RaliContext p_context, RaliImage p_input,
         context->capture_error(e.what());
         ERR(e.what())
     }
-    return input; // Changed to input----------------IMPORTANT
+    return output;
```

Comment:
Why would this change in the new API?

Reply:
This was a bug Rajy , this change is for both old & new API

It did not affect any of our previous trainings because we do not pass the output of CropMirrorNormalize to any other node as input.